### ISSUE TITLE
Assets no longer appearing in CP after caches cleared

### ISSUE THREAD

**Issue by Robert-Black** [statamic/v2-hub#916]
### Steps to reproduce
1. I ran `php please clear:cache` on my production server
2. In the CP, no assets are listed - it says "This folder contains no assets"
3. The data in the folder.yaml files in storage are all intact, and the images are still being served on the site.

### Expected behaviour
After clearing the caches, I expected the assets to still be listed in the Assets section of the CP

### Actual behaviour
Nothing is shown, and there's no message in the logs (other than search updates are failing)

### Server configuration
**Operating system**: OS X 10.6.8

**Web server:** Apache 2.2.24

**PHP version:** PHP 5.6.16

**Statamic version:** 2.1.6

**Updated from an older Statamic or fresh install:** Fresh

**List of installed addons:** Nil

#### Web server error log
```
Insert your webserver log here
```

#### Statamic error log (`/local/storage/logs/statamic.log`)
```
[2016-08-30 05:01:23] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Maximum execution time of 30 seconds exceeded' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index/Term.php:99
Stack trace:
#0 {main}  
[2016-08-30 05:44:38] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Maximum execution time of 30 seconds exceeded' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/File/Filesystem.php:81
Stack trace:
#0 {main}  
[2016-08-30 06:18:17] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Maximum execution time of 30 seconds exceeded' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/File/Filesystem.php:51
Stack trace:
#0 {main}  
[2016-08-30 11:13:24] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Maximum execution time of 30 seconds exceeded' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/File/Filesystem.php:148
Stack trace:
#0 {main}  
[2016-08-30 18:03:39] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function toArray() on null' in /Library/WebServer/Documents/aca/statamic/core/Http/Controllers/SearchController.php:66
Stack trace:
#0 {main}  
[2016-08-30 18:04:42] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Maximum execution time of 30 seconds exceeded' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/File/Filesystem.php:81
Stack trace:
#0 {main}  
[2016-08-30 18:15:30] production.ERROR: exception 'ZendSearch\Lucene\Exception\RuntimeException' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/File/Filesystem.php:55
Stack trace:
#0 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/Directory/Filesystem.php(172): ZendSearch\Lucene\Storage\File\Filesystem->__construct('/Library/WebSer...', 'w+b')
#1 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index/SegmentWriter/AbstractSegmentWriter.php(261): ZendSearch\Lucene\Storage\Directory\Filesystem->createFile('_1.fnm')
#2 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index/SegmentWriter/DocumentWriter.php(198): ZendSearch\Lucene\Index\SegmentWriter\AbstractSegmentWriter->_dumpFNM()
#3 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index/Writer.php(740): ZendSearch\Lucene\Index\SegmentWriter\DocumentWriter->close()
#4 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index.php(1158): ZendSearch\Lucene\Index\Writer->commit()
#5 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index.php(663): ZendSearch\Lucene\Index->commit()
#6 /Library/WebServer/Documents/aca/statamic/vendor/mmanos/laravel-search/src/Mmanos/Search/Index/Zend.php(277): ZendSearch\Lucene\Index->find('xref_id:"72c016...')
#7 /Library/WebServer/Documents/aca/statamic/vendor/mmanos/laravel-search/src/Mmanos/Search/Index/Zend.php(241): Mmanos\Search\Index\Zend->delete('72c016c6-cc0a-4...')
#8 /Library/WebServer/Documents/aca/statamic/core/Search/Search.php(77): Mmanos\Search\Index\Zend->insert('72c016c6-cc0a-4...', Array)
#9 /Library/WebServer/Documents/aca/statamic/core/API/Search.php(60): Statamic\Search\Search->update(NULL)
#10 /Library/WebServer/Documents/aca/statamic/core/Http/Controllers/SearchController.php(36): Statamic\API\Search::update()
#11 [internal function]: Statamic\Http\Controllers\SearchController->update()
#12 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(256): call_user_func_array(Array, Array)
#13 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(164): Illuminate\Routing\Controller->callAction('update', Array)
#14 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(112): Illuminate\Routing\ControllerDispatcher->call(Object(Statamic\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), 'update')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(114): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(69): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(Statamic\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'update')
#21 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(203): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'Statamic\\Http\\C...', 'update')
#22 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(708): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/CP/Authenticate.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 [internal function]: Statamic\Http\Middleware\CP\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#29 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/Outpost.php(52): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 [internal function]: Statamic\Http\Middleware\Outpost->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#32 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/CP/DefaultLocale.php(28): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Statamic\Http\Middleware\CP\DefaultLocale->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#35 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#37 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(710): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#38 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(675): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#39 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#40 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#42 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#43 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/PersistStache.php(22): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Statamic\Http\Middleware\PersistStache->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#46 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#49 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#52 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#54 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#55 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#56 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#57 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#58 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#60 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#61 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/Glide.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#62 [internal function]: Statamic\Http\Middleware\Glide->handle(Object(Illuminate\Http\Request), Object(Closure))
#63 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#64 /Library/WebServer/Documents/aca/statamic/core/StaticCaching/Middleware/Retrieve.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#65 [internal function]: Statamic\StaticCaching\Middleware\Retrieve->handle(Object(Illuminate\Http\Request), Object(Closure))
#66 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#67 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#68 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#69 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#70 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#71 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#72 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#73 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#74 /Library/WebServer/Documents/aca/index.php(156): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#75 {main}

Next exception 'ZendSearch\Lucene\Exception\RuntimeException' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/File/Filesystem.php:55
Stack trace:
#0 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/Directory/Filesystem.php(172): ZendSearch\Lucene\Storage\File\Filesystem->__construct('/Library/WebSer...', 'w+b')
#1 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index/SegmentWriter/AbstractSegmentWriter.php(261): ZendSearch\Lucene\Storage\Directory\Filesystem->createFile('_1.fnm')
#2 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index/SegmentWriter/DocumentWriter.php(198): ZendSearch\Lucene\Index\SegmentWriter\AbstractSegmentWriter->_dumpFNM()
#3 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index/Writer.php(740): ZendSearch\Lucene\Index\SegmentWriter\DocumentWriter->close()
#4 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index.php(1158): ZendSearch\Lucene\Index\Writer->commit()
#5 /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Index.php(429): ZendSearch\Lucene\Index->commit()
#6 /Library/WebServer/Documents/aca/statamic/core/API/Search.php(60): ZendSearch\Lucene\Index->__destruct()
#7 /Library/WebServer/Documents/aca/statamic/core/API/Search.php(60): Statamic\API\Search->update()
#8 /Library/WebServer/Documents/aca/statamic/core/Http/Controllers/SearchController.php(36): Statamic\API\Search::update()
#9 [internal function]: Statamic\Http\Controllers\SearchController->update()
#10 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(256): call_user_func_array(Array, Array)
#11 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(164): Illuminate\Routing\Controller->callAction('update', Array)
#12 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(112): Illuminate\Routing\ControllerDispatcher->call(Object(Statamic\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), 'update')
#13 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#14 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#15 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#16 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(114): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#18 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(69): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(Statamic\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'update')
#19 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(203): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'Statamic\\Http\\C...', 'update')
#20 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#21 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(708): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#23 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/CP/Authenticate.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Statamic\Http\Middleware\CP\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#27 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/Outpost.php(52): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Statamic\Http\Middleware\Outpost->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#30 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/CP/DefaultLocale.php(28): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 [internal function]: Statamic\Http\Middleware\CP\DefaultLocale->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#33 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(710): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(675): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#37 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#38 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#39 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#40 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#41 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/PersistStache.php(22): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 [internal function]: Statamic\Http\Middleware\PersistStache->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#44 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#47 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#50 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#51 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#52 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#53 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#54 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#55 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#56 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#57 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#58 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#59 /Library/WebServer/Documents/aca/statamic/core/Http/Middleware/Glide.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#60 [internal function]: Statamic\Http\Middleware\Glide->handle(Object(Illuminate\Http\Request), Object(Closure))
#61 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#62 /Library/WebServer/Documents/aca/statamic/core/StaticCaching/Middleware/Retrieve.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#63 [internal function]: Statamic\StaticCaching\Middleware\Retrieve->handle(Object(Illuminate\Http\Request), Object(Closure))
#64 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#65 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#66 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#67 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#68 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#69 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#70 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#71 /Library/WebServer/Documents/aca/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#72 /Library/WebServer/Documents/aca/index.php(156): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#73 {main}  
[2016-08-30 18:16:00] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Maximum execution time of 30 seconds exceeded' in /Library/WebServer/Documents/aca/statamic/vendor/zendframework/zendsearch/library/ZendSearch/Lucene/Storage/File/Filesystem.php:51
Stack trace:
#0 {main}  

```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by edalzell** [comment]
@Robert-Black can you increase your PHP timeout setting?

**Comment by Robert-Black** [comment]
Thanks! 

Yep, set it to 600. That fixed the ZendSearch timing out, but **not** the Assets list in CP being blank. Ran `php please clear:cache` again just in case, but still no change.

**Comment by jasonvarga** [comment]
After you update to 2.1.11, can you please confirm if its been fixed?

**Comment by Robert-Black** [comment]
Well… *it's fixed!* :-)

Thank you for following that up! 

Unfortunately, it's not clear to me what was the key factor – after I upgraded to 2.1.11 the whole site crashed, so I reverted with GIT to 2.1.10, at which point I got an error saying the `statamic`folder couldn't be found. So I ran `chmod -R 777 site local statamic assets` but that generated a bunch of errors  (`chmod: Unable to change file mode` for files inside the `local/storage/framework/cache/` folder) so I ran `sudo chmod -R 777 site local statamic assets` and now the assets are showing up correctly in 2.1.10. So all's well that ends well, but I'm not exactly sure what did it – fixing a dumb permissions error? ¯\_(ツ)_/¯
